The job **`mingw-check-tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/10497392696/job/29080154264) [(plain)](https://github.com/rust-lang/rust/commit/9b5a004bf8a388cfe1fc6b844d65e528cca6e2fd/checks/29080154264/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain

COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/

# NOTE: intentionally uses python2 for x.py so we can test it still works.
# validate-toolstate only runs in our CI, so it's ok for it to only support python3.
ENV SCRIPT TIDY_PRINT_DIFF=1 python2.7 ../x.py test \
           --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
# This file is autogenerated by pip-compile with Python 3.10
# by the following command:
#
#    pip-compile --allow-unsafe --generate-hashes reuse-requirements.in
---
#13 2.853 Building wheels for collected packages: reuse
#13 2.854   Building wheel for reuse (pyproject.toml): started
#13 3.099   Building wheel for reuse (pyproject.toml): finished with status 'done'
#13 3.100   Created wheel for reuse: filename=reuse-4.0.3-cp310-cp310-manylinux_2_35_x86_64.whl size=132715 sha256=dfa09868353292d98f811d3efdb0d54d07389e808efc71d68e3b93c514bf8bec
#13 3.100   Stored in directory: /tmp/pip-ephem-wheel-cache-_we88zhs/wheels/3d/8d/0a/e0fc6aba4494b28a967ab5eaf951c121d9c677958714e34532
#13 3.103 Installing collected packages: boolean-py, binaryornot, tomlkit, reuse, python-debian, markupsafe, license-expression, jinja2, chardet, attrs
#13 3.532 Successfully installed attrs-23.2.0 binaryornot-0.4.4 boolean-py-4.0 chardet-5.2.0 jinja2-3.1.4 license-expression-30.3.0 markupsafe-2.1.5 python-debian-0.1.49 reuse-4.0.3 tomlkit-0.13.0
#13 3.533 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#13 4.074 Collecting virtualenv
#13 4.074 Collecting virtualenv
#13 4.124   Downloading virtualenv-20.26.3-py3-none-any.whl (5.7 MB)
#13 4.366      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 5.7/5.7 MB 23.7 MB/s eta 0:00:00
#13 4.427 Collecting filelock<4,>=3.12.2
#13 4.434   Downloading filelock-3.15.4-py3-none-any.whl (16 kB)
#13 4.468 Collecting platformdirs<5,>=3.9.1
#13 4.476   Downloading platformdirs-4.2.2-py3-none-any.whl (18 kB)
#13 4.498 Collecting distlib<1,>=0.3.7
#13 4.517      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 468.9/468.9 KB 47.8 MB/s eta 0:00:00
#13 4.517      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 468.9/468.9 KB 47.8 MB/s eta 0:00:00
#13 4.599 Installing collected packages: distlib, platformdirs, filelock, virtualenv
#13 4.787 Successfully installed distlib-0.3.8 filelock-3.15.4 platformdirs-4.2.2 virtualenv-20.26.3
#13 DONE 4.9s

#14 [7/8] COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
#14 DONE 0.0s
---
DirectMap4k:      194496 kB
DirectMap2M:     6096896 kB
DirectMap1G:    12582912 kB
##[endgroup]
Executing TIDY_PRINT_DIFF=1 python2.7 ../x.py test            --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
+ TIDY_PRINT_DIFF=1 python2.7 ../x.py test --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint,cpp:fmt
    Finished `dev` profile [unoptimized] target(s) in 0.04s
##[endgroup]
downloading https://ci-artifacts.rust-lang.org/rustc-builds-alt/6b678c57b63b3062fb97130b3617b82657f59c80/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz
extracting /checkout/obj/build/cache/llvm-6b678c57b63b3062fb97130b3617b82657f59c80-true/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz to /checkout/obj/build/x86_64-unknown-linux-gnu/ci-llvm
---
fmt check
Diff in /checkout/library/core/src/num/int_macros.rs:1313:
         }
 
         /// Unbounded shift left. Computes `self << rhs`, without bounding the value of `rhs`
-        /// 
-        /// If `rhs` is larger or equal to the number of bits in `self`, 
+        ///
+        /// If `rhs` is larger or equal to the number of bits in `self`,
         /// the entire value is shifted out, and `0` is returned.
         /// # Examples
Diff in /checkout/library/core/src/num/int_macros.rs:1321:
-        /// 
+        ///
+        ///
         /// Basic usage:
         /// ```
         #[doc = concat!("assert_eq!(0x1", stringify!($SelfT), ".unbounded_shl(4), 0x10);")]
Diff in /checkout/library/core/src/num/int_macros.rs:1438:
 
 
         /// Unbounded shift right. Computes `self >> rhs`, without bounding the value of `rhs`
-        /// 
-        /// If `rhs` is larger or equal to the number of bits in `self`, 
+        ///
+        /// If `rhs` is larger or equal to the number of bits in `self`,
         /// the entire value is shifted out, which yields `0` for a positive number,
         /// and `-1` for a negative number.
Diff in /checkout/library/core/src/num/int_macros.rs:1446:
         /// # Examples
-        /// 
+        ///
+        ///
         /// Basic usage:
         /// ```
         #[doc = concat!("assert_eq!(0x10", stringify!($SelfT), ".unbounded_shl(4), 0x1);")]
Diff in /checkout/library/core/src/num/uint_macros.rs:1502:
 
 
                 /// Unbounded shift left. Computes `self << rhs`, without bounding the value of `rhs`
-        /// 
-        /// If `rhs` is larger or equal to the number of bits in `self`, 
+        ///
+        /// If `rhs` is larger or equal to the number of bits in `self`,
         /// the entire value is shifted out, and `0` is returned.
         /// # Examples
Diff in /checkout/library/core/src/num/uint_macros.rs:1510:
-        /// 
+        ///
+        ///
         /// Basic usage:
         /// ```
         #[doc = concat!("assert_eq!(0x1", stringify!($SelfT), ".unbounded_shl(4), 0x10);")]
Diff in /checkout/library/core/src/num/uint_macros.rs:1627:
 
 
         /// Unbounded shift right. Computes `self >> rhs`, without bounding the value of `rhs`
-        /// 
-        /// If `rhs` is larger or equal to the number of bits in `self`, 
+        ///
+        /// If `rhs` is larger or equal to the number of bits in `self`,
         /// the entire value is shifted out, and `0` is returned.
         /// # Examples
Diff in /checkout/library/core/src/num/uint_macros.rs:1635:
-        /// 
+        ///
+        ///
         /// Basic usage:
         /// ```
         #[doc = concat!("assert_eq!(0x10", stringify!($SelfT), ".unbounded_shr(4), 0x10);")]
fmt error: Running `"/checkout/obj/build/x86_64-unknown-linux-gnu/rustfmt/bin/rustfmt" "--config-path" "/checkout" "--edition" "2021" "--unstable-features" "--skip-children" "--check" "/checkout/src/tools/rustc-perf-wrapper/src/config.rs" "/checkout/src/tools/bump-stage0/src/main.rs" "/checkout/library/core/src/num/flt2dec/decoder.rs" "/checkout/library/core/src/num/flt2dec/strategy/dragon.rs" "/checkout/library/core/src/num/flt2dec/strategy/grisu.rs" "/checkout/library/core/src/num/flt2dec/mod.rs" "/checkout/library/core/src/num/flt2dec/estimator.rs" "/checkout/library/core/src/num/int_macros.rs" "/checkout/library/core/src/num/error.rs" "/checkout/library/core/src/num/diy_float.rs" "/checkout/src/tools/miropt-test-tools/src/lib.rs" "/checkout/src/tools/lint-docs/src/main.rs" "/checkout/src/tools/lint-docs/src/groups.rs" "/checkout/src/tools/lint-docs/src/lib.rs" "/checkout/library/core/src/num/shells/int_macros.rs" "/checkout/library/core/src/num/shells/isize.rs" "/checkout/library/core/src/num/shells/i128.rs" "/checkout/library/core/src/num/shells/u128.rs" "/checkout/library/core/src/num/shells/u32.rs" "/checkout/library/core/src/num/shells/u16.rs" "/checkout/library/core/src/num/shells/i16.rs" "/checkout/library/core/src/num/shells/u64.rs" "/checkout/library/core/src/num/shells/u8.rs" "/checkout/library/core/src/num/shells/i8.rs" "/checkout/library/core/src/num/shells/usize.rs" "/checkout/library/core/src/num/shells/i32.rs" "/checkout/library/core/src/num/shells/i64.rs" "/checkout/library/core/src/num/bignum.rs" "/checkout/library/core/src/num/f128.rs" "/checkout/library/core/src/num/mod.rs" "/checkout/library/core/src/num/uint_macros.rs" "/checkout/library/core/src/num/wrapping.rs" "/checkout/library/core/src/num/f32.rs" "/checkout/src/tools/lld-wrapper/src/main.rs" "/checkout/library/core/src/num/dec2flt/table.rs" "/checkout/library/core/src/num/dec2flt/decimal.rs" "/checkout/library/core/src/num/dec2flt/float.rs" "/checkout/library/core/src/num/dec2flt/mod.rs" "/checkout/library/core/src/num/dec2flt/common.rs" "/checkout/library/core/src/num/dec2flt/lemire.rs" "/checkout/library/core/src/num/dec2flt/fpu.rs" "/checkout/library/core/src/num/dec2flt/slow.rs" "/checkout/library/core/src/num/dec2flt/parse.rs" "/checkout/library/core/src/num/dec2flt/number.rs" "/checkout/library/core/src/num/f16.rs" "/checkout/library/core/src/num/int_log10.rs" "/checkout/library/core/src/num/nonzero.rs" "/checkout/library/core/src/num/overflow_panic.rs" "/checkout/library/core/src/num/saturating.rs" "/checkout/library/core/src/num/fmt.rs" "/checkout/library/core/src/num/f64.rs" "/checkout/src/tools/generate-copyright/src/main.rs" "/checkout/src/tools/generate-copyright/src/cargo_metadata.rs" "/checkout/library/core/src/ptr/metadata.rs" "/checkout/library/core/src/ptr/mod.rs" "/checkout/library/core/src/ptr/unique.rs" "/checkout/library/core/src/ptr/const_ptr.rs" "/checkout/library/core/src/ptr/mut_ptr.rs" "/checkout/library/core/src/ptr/non_null.rs" "/checkout/library/core/src/ptr/alignment.rs" "/checkout/src/tools/replace-version-placeholder/src/main.rs" "/checkout/library/rustc-std-workspace-std/lib.rs" "/checkout/library/unwind/src/unwinding.rs" "/checkout/src/tools/rustc-perf-wrapper/src/main.rs"` failed.
If you're running `tidy`, try again with `--bless`. Or, if you just want to format code, run `./x.py fmt` instead.
  local time: Wed Aug 21 21:05:49 UTC 2024
  network time: Wed, 21 Aug 2024 21:05:50 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/library/core/src/num/int_macros.rs
+++ b/library/core/src/num/int_macros.rs
@@ -1313,12 +1313,12 @@ macro_rules! int_impl {
         }
 
         /// Unbounded shift left. Computes `self << rhs`, without bounding the value of `rhs`
-        /// 
-        /// If `rhs` is larger or equal to the number of bits in `self`, 
+        ///
+        /// If `rhs` is larger or equal to the number of bits in `self`,
         /// the entire value is shifted out, and `0` is returned.
         ///
         /// # Examples
-        /// 
+        ///
         /// Basic usage:
         /// ```
         #[doc = concat!("assert_eq!(0x1", stringify!($SelfT), ".unbounded_shl(4), 0x10);")]
@@ -1438,13 +1438,13 @@ macro_rules! int_impl {
         }
 
         /// Unbounded shift right. Computes `self >> rhs`, without bounding the value of `rhs`
-        /// 
-        /// If `rhs` is larger or equal to the number of bits in `self`, 
+        ///
+        /// If `rhs` is larger or equal to the number of bits in `self`,
         /// the entire value is shifted out, which yields `0` for a positive number,
         /// and `-1` for a negative number.
         ///
         /// # Examples
-        /// 
+        ///
         /// Basic usage:
         /// ```
         #[doc = concat!("assert_eq!(0x10", stringify!($SelfT), ".unbounded_shl(4), 0x1);")]

--- a/library/core/src/num/int_macros.rs
+++ b/library/core/src/num/int_macros.rs
@@ -1330,11 +1330,11 @@ macro_rules! int_impl {
                       without modifying the original"]
         #[inline]
         pub const fn unbounded_shl(self, rhs: u32) -> $SelfT{
-            if rhs < Self::BITS{
+            if rhs < Self::BITS {
                 // SAFETY:
                 // rhs is just checked to be in-range above
                 unsafe { self.unchecked_shl(rhs) }
-            }else{
+            } else {
                 0
             }
         }
@@ -1457,11 +1457,11 @@ macro_rules! int_impl {
                       without modifying the original"]
         #[inline]
         pub const fn unbounded_shr(self, rhs: u32) -> $SelfT{
-            if rhs < Self::BITS{
+            if rhs < Self::BITS {
                 // SAFETY:
                 // rhs is just checked to be in-range above
                 unsafe { self.unchecked_shr(rhs) }
-            }else{
+            } else {
                 // A shift by `Self::BITS-1` suffices for signed integers, because the sign bit is copied for each of the shifted bits.
 
                 // SAFETY:

--- a/library/core/src/num/int_macros.rs
+++ b/library/core/src/num/int_macros.rs
@@ -1329,11 +1329,11 @@ macro_rules! int_impl {
         #[must_use = "this returns the result of the operation, \
                       without modifying the original"]
         #[inline]
-        pub const fn unbounded_shl(self, v: u32) -> $SelfT{
-            if v < Self::BITS{
+        pub const fn unbounded_shl(self, rhs: u32) -> $SelfT{
+            if rhs < Self::BITS{
                 // SAFETY:
-                // v is just checked to be in-range above
-                unsafe{self.unchecked_shl(v)}
+                // rhs is just checked to be in-range above
+                unsafe { self.unchecked_shl(rhs) }
             }else{
                 0
             }
@@ -1456,17 +1456,17 @@ macro_rules! int_impl {
         #[must_use = "this returns the result of the operation, \
                       without modifying the original"]
         #[inline]
-        pub const fn unbounded_shr(self, v: u32) -> $SelfT{
-            if v < Self::BITS{
+        pub const fn unbounded_shr(self, rhs: u32) -> $SelfT{
+            if rhs < Self::BITS{
                 // SAFETY:
-                // v is just checked to be in-range above
-                unsafe{self.unchecked_shr(v)}
+                // rhs is just checked to be in-range above
+                unsafe { self.unchecked_shr(rhs) }
             }else{
                 // A shift by `Self::BITS-1` suffices for signed integers, because the sign bit is copied for each of the shifted bits.
 
                 // SAFETY:
                 // `Self::BITS-1` is guaranteed to be less than `Self::BITS`
-                unsafe{self.unchecked_shr(Self::BITS - 1)}
+                unsafe { self.unchecked_shr(Self::BITS - 1) }
             }
         }
 

--- a/library/core/src/num/uint_macros.rs
+++ b/library/core/src/num/uint_macros.rs
@@ -1518,11 +1518,11 @@ macro_rules! uint_impl {
         #[must_use = "this returns the result of the operation, \
                       without modifying the original"]
         #[inline]
-        pub const fn unbounded_shl(self, v: u32) -> $SelfT{
-            if v < Self::BITS{
+        pub const fn unbounded_shl(self, rhs: u32) -> $SelfT{
+            if rhs < Self::BITS{
                 // SAFETY:
-                // v is just checked to be in-range above
-                unsafe{self.unchecked_shl(v)}
+                // rhs is just checked to be in-range above
+                unsafe { self.unchecked_shl(rhs) }
             }else{
                 0
             }
@@ -1643,11 +1643,11 @@ macro_rules! uint_impl {
         #[must_use = "this returns the result of the operation, \
                       without modifying the original"]
         #[inline]
-        pub const fn unbounded_shr(self, v: u32) -> $SelfT{
-            if v < Self::BITS{
+        pub const fn unbounded_shr(self, rhs: u32) -> $SelfT{
+            if rhs < Self::BITS{
                 // SAFETY:
-                // v is just checked to be in-range above
-                unsafe{self.unchecked_shr(v)}
+                // rhs is just checked to be in-range above
+                unsafe { self.unchecked_shr(rhs) }
             }else{
                 0
             }

--- a/library/core/src/num/uint_macros.rs
+++ b/library/core/src/num/uint_macros.rs
@@ -1519,11 +1519,11 @@ macro_rules! uint_impl {
                       without modifying the original"]
         #[inline]
         pub const fn unbounded_shl(self, rhs: u32) -> $SelfT{
-            if rhs < Self::BITS{
+            if rhs < Self::BITS {
                 // SAFETY:
                 // rhs is just checked to be in-range above
                 unsafe { self.unchecked_shl(rhs) }
-            }else{
+            } else {
                 0
             }
         }
@@ -1644,11 +1644,11 @@ macro_rules! uint_impl {
                       without modifying the original"]
         #[inline]
         pub const fn unbounded_shr(self, rhs: u32) -> $SelfT{
-            if rhs < Self::BITS{
+            if rhs < Self::BITS {
                 // SAFETY:
                 // rhs is just checked to be in-range above
                 unsafe { self.unchecked_shr(rhs) }
-            }else{
+            } else {
                 0
             }
         }

--- a/library/core/src/num/uint_macros.rs
+++ b/library/core/src/num/uint_macros.rs
@@ -1502,12 +1502,12 @@ macro_rules! uint_impl {
         }
 
                 /// Unbounded shift left. Computes `self << rhs`, without bounding the value of `rhs`
-        /// 
-        /// If `rhs` is larger or equal to the number of bits in `self`, 
+        ///
+        /// If `rhs` is larger or equal to the number of bits in `self`,
         /// the entire value is shifted out, and `0` is returned.
         ///
         /// # Examples
-        /// 
+        ///
         /// Basic usage:
         /// ```
         #[doc = concat!("assert_eq!(0x1", stringify!($SelfT), ".unbounded_shl(4), 0x10);")]
@@ -1627,12 +1627,12 @@ macro_rules! uint_impl {
         }
 
         /// Unbounded shift right. Computes `self >> rhs`, without bounding the value of `rhs`
-        /// 
-        /// If `rhs` is larger or equal to the number of bits in `self`, 
+        ///
+        /// If `rhs` is larger or equal to the number of bits in `self`,
         /// the entire value is shifted out, and `0` is returned.
         ///
         /// # Examples
-        /// 
+        ///
         /// Basic usage:
         /// ```
         #[doc = concat!("assert_eq!(0x10", stringify!($SelfT), ".unbounded_shr(4), 0x10);")]

